commons-dlp/src/main/java/org/exoplatform/commons/dlp/rest/DlpRestServices.java

```diff
@@ -0,0 +1,83 @@
+package org.exoplatform.commons.dlp.rest;
+
+import org.exoplatform.common.http.HTTPStatus;
+import org.exoplatform.commons.api.settings.ExoFeatureService;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.services.organization.Membership;
+import org.exoplatform.services.organization.OrganizationService;
+import org.exoplatform.services.rest.resource.ResourceContainer;
+import org.exoplatform.services.security.ConversationState;
+
+import io.swagger.annotations.*;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Collection;
+
+@Path("/features/dlp")
```

Comment:
I think that /dlp is sufficient here, WDYT @rdenarie @ahmadmido ?

Reply:
Ok (y)

Reply:
:+1: